docs/TESTS.md

```diff
@@ -4,9 +4,17 @@ We recommend you install [pytest](http://pytest.org/latest/) and
 [pytest-cache](http://pythonhosted.org/pytest-cache/). `pytest` is a testing
 tool that will give you more flexibility over running your unit tests.
 
+If you want to install the version of pytest that obeys Python 2.X syntax, then use `pip`:
+
 ```bash
 pip install pytest pytest-cache
 ```
+If you instead want the vesrion of pytest that obeys Python 3.X syntax, then use `pip3`:
+
+```bash
+pip3 install pytest pytest-cache
+```
+**Note:** Whichever version of pytest you install first will be the default one used whenever `pytest` is executed, regardless of whether you have installed both versions.
```

Comment:
It might be worth mentioning that you can check the default version by running `pytest --version` (on Linux and presumably OSX?, not sure what the equivalent is in Windows).

On my system it returns:
```shell
$ pytest --version
This is pytest version 3.2.1, imported from /usr/lib/python3.6/site-packages/pytest.py
```